ice_discharge.org

```diff
@@ -5191,70 +4756,29 @@ df
 #+END_SRC
 
 #+RESULTS:
-:RESULTS:
 #+begin_example
 D SE described:
  count     38.000000
-mean     138.768054
-std        6.316258
-min      125.883129
-25%      133.361429
-50%      138.973207
-75%      143.447892
-max      150.430592
+mean     138.291279
```

Comment:
This section and around here is digging into SE, because in the original paper we had large disagreement with Enderlin in the SE.

Now we should have a comparison against an earlier version of our product. Unfortunately, no figures or tables were provided in #39, but there may have been emails with more detail?

Reply:
I have a comparison for the entire Greenland Ice Sheet from 2020, it was create a while back but I think it illustrates the issue well, the label dataverse refers to the version not using PROMICE IV. Where should this go?
![GIS_compare](https://github.com/GEUS-Glaciology-and-Climate/ice_discharge/assets/71708740/b4a669ca-1894-4be9-9e67-ab320fc7547d)